test/check-application

```diff
@@ -2918,9 +2915,12 @@ class TestApplication(testlib.MachineCase):
     def _testRenameContainer(self, auth):
         b = self.browser
         container_name = "rename"
+        service_name = "quadlet"
         container_name_new = "rename-new"
 
         self.execute(auth, f"podman container create -t --name {container_name} {IMG_BUSYBOX}")
+        container_label = "--label PODMAN_SYSTEMD_UNIT=rename.service"
```

Comment:
Wouldn't it be nicer to test with a real quadlet instead of making our own low budget quadlet?

Reply:
Yes, let's please test this end to end -- this is internal stuff that may change or get broken over time.

Reply:
Sure, can easily make one. Will do :)